pkg/api/v1beta3/conversion.go

```diff
@@ -20,11 +20,2638 @@ import (
 	"fmt"
 
 	newer "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
 )
 
 func init() {
+	err := newer.Scheme.AddConversionFuncs(
```

Comment:
It is correct that these are auto-generated, yes? In that case I do NOT think we should commit in this way. The reason is that the other conversion functions are manually written, and if we do this we'll have a mix of manually and automatically written functions.

So, my recommendation is that you separate this in some way. E.g., autogenerate a ```func AddGeneratedConversionFunctions()```, and call that first in init (and then it can be overridden with any manually generate functions.

I think this is very important because eventually the flow should go the other way-- you should produce the auto-generated stuff by gluing together the manually written stuff. So it's important to make the difference between the auto-generated and manually written code very obvious.